bot/src/wrapper/_communication.py

```diff
@@ -268,3 +379,64 @@ async def caller(session: aiohttp.ClientSession) -> UserIdExperiencePatchOutput:
                 raise UnknownNetworkError(message)
 
         return await make_request(session, caller)
+
+    @staticmethod
+    async def set_user_experience(
+        session: aiohttp.ClientSession, user_id: int, src: UserIdExperiencePostInput
+    ) -> UserIdExperiencePostOutput:
+        """Set the user experience by user id through a direct HTTP request."""
+
+        async def caller(session: aiohttp.ClientSession) -> UserIdExperiencePatchOutput:
+            async with session.post(f"user/{user_id}/experience/set", json=src) as resp:
+                if resp.ok:
+                    return await resp.json()
+                if resp.status == Status.NOT_FOUND:
+                    message = f"User with user id {user_id} cannot be found"
+                    raise DoesNotExistError(message)
+                message = (
+                    "Undefined behaviour bot.src.wrapper.UserRawAPI.set_user_experience,"
+                    f" Status received {resp.status}"
+                )
+                raise UnknownNetworkError(message)
+
+        return await make_request(session, caller)
+
+
+class AchievementRawAPI:
+    @staticmethod
+    async def get_achievements(session: aiohttp.ClientSession) -> AchievementGetOutput:
+        """Get all achievements through a direct HTTP request."""
+
+        async def caller(session: aiohttp.ClientSession) -> AchievementGetOutput:
+            async with session.get("/achievements") as resp:
```

Comment:
KK, think I have it as `achievement` in the API side, but will change it. This will be fine

Reply:
Fixed on API side.